spec/comment_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_0BxGuVvo', bio: 'Teacher from Mexico.')
+  post = Post.create(user:, title: 'Hello', text: 'This is my first post')
+
+  subject { Comment.new(user:, post:, text: 'Comment text') }
+
+  before { subject.save }
+
+  it 'should increment post comments counter' do
+    counter = post.comments_counter
+    subject.save
+    expect(post.comments_counter).to eq(counter)
+  end
+end
```

Comment:
- [ ] Let's add some tests for your model variations.
For example 
```ruby
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```

Reply:
Added validation